src/Components/Auth/SignIn/SignIn.scss

```diff
@@ -0,0 +1,206 @@
+$blue-text-color: #2a96cc;
+$oauth-text-color: hsla(120, 100%, 0%, 0.302);
+$box-shadow-color: hsla(0, 0%, 0%, 0.102);
+$button-background-color: #2e95c9;
+$button-text-color: #f7f7f7;
+$input-text-color: hsla(0, 0%, 0%, 0.561);
+$input-bottom-border-color: hsla(0, 0%, 0%, 0.329);
+$font-family: "Roboto",
+sans-serif;
+$error-message-text-color: #a33e10;
+
+.main-signin {
+  height: 100%;
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: 60%;
+}
+
+.signin-section {
+  display: flex;
+  flex-direction: column;
+  height: 563px;
+  margin: 30px auto;
+  padding: 20px;
+  position: relative;
+  text-align: center;
+  top: 5%;
+  width: 50%;
+
+  &__h1 {
```

Comment:
You change this class name to another one because **h1** is a reserved keyword.

Reply:
Feedback implemented

Reply:
Not yet. Remove all `&__h1` please.

Reply:
Not yet.